docs/admin-backup-and-restore.md

```diff
@@ -123,14 +175,32 @@ The default location is the `BackupStorageLocation` object with `.spec.default`
 For example, you can create a `ManagementBackup` object that backs up to the storage object created in the previous step
 every 6 minutes would look like this:
```

Comment:
Moshiur, I realise this wasn't a part of your PR, but as long as we're here....

I don't believe `every 6 minutes` is correct, based on the example that follows (and the URL I'm including in the edit). Rather, this should say `every 6 hours`.

Suggested edit:

```
For example, you can create a `ManagementBackup` object that backs up to the storage object
created in the previous step every 6 hours
(ref: [Kubernetes CronJob schedule syntax, "Vixie cron" step values](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#schedule-syntax))
would look like this:
```

If you'd prefer, I can file a separate PR for this.